czarjulius/Banka-React
# #168096737 Implement user create bank account

#### What does this PR do?
- 

#### Description of Task to be completed?
- Create open account action and type
- Create open account reducer and connect to store
- Add logout functionality

#### How should this be manually tested?
- Logged in user should click on `Create Account` on the sidebar at the user dashboard.
- Enter the account type and submit.

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
- [#168096737](https://www.pivotaltracker.com/story/show/168096737)

#### Screenshots (if appropriate)
-N/A

#### Questions:
- N/A
